DB Setup/createDB.sql

```diff
@@ -0,0 +1,116 @@
+create database VA_Trial;
+\c va_trial
+create table SessionKeys (
+	SessionKeyID int, 
+	UserID int, 
+	CreatedTime Time,
+	LastUsedTime Time,
+	PRIMARY KEY (SessionKeyID)
+);
+
+create table tfiSurvey ( 
+	tfiSurveyID int NOT NULL, 
+	PatientID int, 
+	CompletionDate Date,
+	CompletionTime Time,
+	TFI_I int,
+	TFI_Sc int, 
+	TFI_C int, 
+	TFI_SI int,
+	TFI_A int,
+	TFI_R int, 
+	TFI_Q int, 
+	TFI_E int, 
+	TFI_OverallScore int,
+	PRIMARY KEY (tfiSurveyID) 
+);
+
+create table thsSurvey ( 
+	thsSurveyID int NOT NULL, 
+	PatientID int, 
+	CompletionDate Date,
+	CompletionTime Time,
+	THS_SectionA int,
+	THS_SectionB int, 
+	THS_SectionC text,
+	PRIMARY KEY (thsSurveyID)
+);
+
+create table tsSurvey ( 
+	tsSurveyID int NOT NULL, 
+	PatientID int, 
+	CompletionDate Date,
+	CompletionTime Time,
+	TS_Type text,
+	PRIMARY KEY (tsSurveyID)
+);
+
+create table Patient (
+	PatientID int NOT NULL,
+	FirstName text,
+	LastName text,
```

Comment:
To my knowledge, we can't and arn't storing PII or HIPPA sensitive data. I think we need to remove first name last name.